set.mm

```diff
@@ -139476,7 +139491,11 @@ notation is used by Donald Knuth for iterated exponentiation (_Science_
        4-Jun-2014:  The definition was extended to include negative integer
        exponents.  For example, ` ( -u 3 ^ -u 2 ) = ( 1 / 9 ) ` ( ~ ex-exp ).
        The case ` x = 0 , y < 0 ` gives the value ` ( 1 / 0 ) ` , so we will
-       avoid this case in our theorems.  (Contributed by Raph Levien,
+       avoid this case in our theorems.
+
+       For a definition of exponentiation including complex exponents see
+       ~ df-cxp (complex power function).  Both definitions are equivalent for
```

Comment:
(complex exponentiation function)

Reply:
I used "complex exponentiation" in my latest commit, because `^c` is not a function, but an operation.